lib/http-server.js

```diff
@@ -54,13 +54,18 @@ function HttpServer(options) {
     options.cache === -1 ? 'no-cache, no-store, must-revalidate' :
     options.cache // in seconds.
   );
-  this.showDir = options.showDir !== 'false';
-  this.autoIndex = options.autoIndex !== 'false';
-  this.showDotfiles = options.showDotfiles;
-  this.gzip = options.gzip === true;
-  this.brotli = options.brotli === true;
+  if(options.showDir)
+    this.showDir = options.showDir.toLowerCase() !== 'false';
+  if(options.autoIndex)
+    this.autoIndex = options.autoIndex.toLowerCase() !== 'false';
```

Comment:
If we only set these values if the option is truthy, will these values be `undefined` when we use them later?

Reply:
All these options are strings, so these `if conditions` will be true if the options are non empty strings. If these options are not entered then they are `undefined` by default.

Reply:
These were  `undefined` even before this changes (when that option is empty)

Reply:
If you look farther down, when we pass these options into the server, they're expected to be defined. By putting these in the `if`, they will sometimes not be defined